docs/src/pages/system/advanced/StyleFunctionSxDemo.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import styled, { InterpolationFunction, ThemeProvider } from 'styled-components';
+import { unstable_styleFunctionSx } from '@material-ui/system';
+import NoSsr from '@material-ui/core/NoSsr';
+import { SxProps } from '@material-ui/core/Box';
```

Comment:
From my perspective, this breaks the purpose of this demo: allowing to use the system without `@material-ui/core`.

```suggestion
import { SxProps } from '@material-ui/system';
```

Reply:
Will be done as part of https://github.com/mui-org/material-ui/pull/23599